components/autofill/src/sync/address/incoming.rs

```diff
@@ -27,9 +31,16 @@ pub fn stage_incoming(
     let mut incoming_tombstones = Vec::with_capacity(incoming_payloads.len());
 
     for payload in incoming_payloads {
+        log::trace!(
+            "incoming payload {} (deleted={})",
+            payload.id,
+            payload.deleted
+        );
         match payload.deleted {
-            true => incoming_tombstones.push(payload.into_record::<Record>().unwrap()),
-            false => incoming_records.push(payload.into_record::<Record>().unwrap()),
+            // XXX - unwraps below should be removed. We probably want to handle
```

Comment:
Actually, I think these are your comments to me before my abrupt handoff of this PR. I can make these changes in a cleanup commit.